src/components/proofs/EmailProof.tsx

```diff
@@ -22,10 +18,8 @@ import classnames, {
   gradientColorStops,
   justifyContent,
   lineHeight,
-  margin,
   space,
   textColor,
```

Comment:
Not a good solution to use 'text-' classes here, outside Text component. May we avoid it?

Reply:
We will add a separate task)